app/steps/medicaid/contact_home_address.rb

```diff
@@ -2,7 +2,10 @@ module Medicaid
   class ContactHomeAddress < Step
     step_attributes(
       :street_address,
+      :street_address_2,
       :city,
+      :county,
```

Comment:
now that county and state are attributes on the step, what about adding validations for them at the step level so we can be sure they aren't left blank?

Reply:
Thanks for pointing this out. @hartsick and I will address this in the work on non-Genesee counties:

https://www.pivotaltracker.com/story/show/152981944